### PR TITLE
13.0 mig hr holidays leave repeated

### PR DESCRIPTION
**Pull request by hieulucky111**
Migration module `hr_holidays_leave_repeated` to version 13.0

### CODE REVIEW

**Review by jcdrubay — APPROVED** [review]

**Comment by astirpe** [comment]
Can you cherry-pick commit https://github.com/OCA/hr/pull/863/commits/87f9aa98d5045b6318d809277d2200bcadef4d3e from PR https://github.com/OCA/hr/pull/863 ?

File: hr_holidays_leave_repeated/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2016-2019 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "HR Holidays leave repeated",
+    "summary": "Define periodical leaves",
+    "author": "Onestein, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/hr/",
```
Comment (i-vyshnevska) [inline hr_holidays_leave_repeated/__manifest__.py:8]:
```suggestion
    "website": "https://github.com/OCA/hr-holidays/",
```

File: hr_holidays_leave_repeated/models/hr_leave.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                ),
+            },
+        }
+
+    @api.model
+    def _update_repeated_leave_vals(self, vals, employee):
+        vals_dict = self._get_repeated_vals_dict()
+        param_dict = vals_dict[vals.get("repeat_every")]
+        from_dt = fields.Datetime.from_string(vals.get("date_from"))
+        to_dt = fields.Datetime.from_string(vals.get("date_to"))
```
Comment (i-vyshnevska) [inline hr_holidays_leave_repeated/models/hr_leave.py:92]: suppose dates here already in datetime format

**Pull request merged**